lib/core/src/lib/ui-header/navbar/navbar-item.component.md

```diff
@@ -0,0 +1,83 @@
+# Navbar Item Component
+
+`standalone`, `component`
+
+The Navbar Item component is a subcomponent of the [Navbar](./navbar.component.md) component, designed to represent individual navigation links or actions within the Navbar. It can be used to navigate to different sections of the application or to perform specific actions.
+
+## Usage
+
+Adding a navbar item to navigate to the home page:
+
+```html
+<adf-navbar-item routerLink="/home" label="Home"></adf-navbar-item>
+```
+
+Adding a navbar item that triggers an action:
+
+```html
+<adf-navbar-item (click)="doSomething()" label="Action"></adf-navbar-item>
```

Comment:
Just curious, but what would happen if one was to do something like 
```
<adf-navbar-item (click)="doSomething()" routerLink="/home" label="Action"></adf-navbar-item>
```
There is no custom click event in navbar-item. So I guess its working on the native HTML click event. In the case above, would it do something like route to the provided link, and perform the click action as well?

Reply:
I think if correct route was provided, then only navigation will be performed